mlrun/model_monitoring/controller.py

```diff
@@ -415,12 +415,12 @@ def model_endpoint_process(
                 )
 
                 for start_infer_time, end_infer_time in batch_window.get_intervals():
-                    prediction_metric = tsdb_connector.read_predictions(
-                        endpoint_id=endpoint_id,
-                        start=start_infer_time,
-                        end=end_infer_time,
+                    df = m_fs.to_dataframe(
```

Comment:
I assume you tested it in Iguazio and it works, but will it work without the `storage_options` in CE?